mlrun/runtimes/databricks_runtime_wrap.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+def run_mlrun_databricks_job(
+    context,
+    internal_handler,
+    is_local_code=True,
+    token_key="DATABRICKS_TOKEN",
+    **kwargs,
+):
+    import ast
+    import datetime
+    import json
+    import os
+    import shutil
+    import tempfile
+    import uuid
+
+    import astor
+    from databricks.sdk import WorkspaceClient
+    from databricks.sdk.service.compute import ClusterSpec
+    from databricks.sdk.service.jobs import Run, SparkPythonTask, SubmitTask
+
+    import mlrun
+
+    project_name = context.project
+    project = mlrun.get_or_create_project(project_name)
+    get_secret_func = project.get_secret
+    logger = context.logger
+
+    def get_modified_code(function_name, script_file, handler):
+        with open(script_file, "r") as file:
+            tree = ast.parse(file.read())
+
+        # Find the function node to delete
+        to_delete = None
+        for node in tree.body:
+            if isinstance(node, ast.FunctionDef) and node.name == function_name:
+                to_delete = node
```

Comment:
```suggestion
                tree.body.remove(node)
```
You just need to delete and exit the for-loop. Why do you need the `to_delete` indication and delete after the loop?